regulations/templates/regulations/search.html

```diff
@@ -41,6 +41,7 @@
                         {% for result in results %}
                             {% if result.label|length > 1 %}
                             <div class="result">
+                                <!--{{ result.rank }}-->
```

Comment:
should this be here?

Reply:
I was thinking it could be a nice way for us to debug / monitor the search behavior? It shows up if you view the source of the page